src/base-object.h

```diff
@@ -44,7 +44,7 @@ class BaseObject {
   static inline void WeakCallback(
       const v8::WeakCallbackInfo<Type>& data);
 
-  v8::Persistent<v8::Object> handle_;
+  v8::Persistent<v8::Object> objecthandle_;
```

Comment:
It would be infinitesimally more idiomatic to call it `object_handle_` or `persistent_handle_` or just `object_`.  Not a blocker, though.